config.yml

```diff
@@ -1,5 +0,0 @@
-public_dir: "./public"
```

Comment:
I would still like to keep this one to show users what configs are there. It is also part documentation.

Reply:
How about `config.reference.yml`?

Reply:
++. Should we fall back to it or just have it there?

Reply:
There won't be any `config.yml` by default as it's not required, but there is will be a `config.reference.yml` if users would like to preview the configuration file.

Reply:
SGTM

Reply:
Fixed